examples/plot_simulate_evoked.py

```diff
@@ -81,23 +126,32 @@
 print('Mean spike rates for individual trials:')
 print(trial_rates)
 
-
 ###############################################################################
 # Now, let us try to make the exogenous driving inputs to the cells
-# synchronous and see what happens
+# synchronous and see what happens. This is achieved by setting the parameter
+# ``sync_within_trial`` to ``True``. Using the ``copy``-method, we can create
+# a clone of the network defined above, and then modify the drive dynamics for
+# each drive.
+
+net_sync = net.copy()
+net_sync.external_drives['evdist1']['dynamics']['sync_within_trial'] = True
```

Comment:
Future PR idea, function to sync all external drives automatically?

Reply:
Sure :) I just don't have an idea of how much this syncing business is actually used?

Reply:
The main benefit would be teasing apart the contribution of specific inputs. Namely, holding every drive fixed except for one that is allowed to vary.